### PR TITLE
Pass auth tests

### LINKED ISSUE #118: User Story 6, Users cannot navigate to certain paths
```
Users should see a 404 error under the following conditions:
- if visitors try to navigate to any /profile path
- if visitors try to navigate to any /dashboard path
- if visitors try to navigate to any /admin path
- if registered users try to navigate to any /dashboard path
- if registered users try to navigate to any /admin path
- if merchants try to navigate to any /profile path
- if merchants try to navigate to any /admin path
- if merchants try to navigate to any /cart path
- if admin users try to navigate to any /profile path
- if admin users try to navigate to any /dashboard path
- if admin users try to navigate to any /cart path

If you think of any additional pages to block, please do so.
```

DO THESE STORIES FIRST:
- User Story 2, Visitor Navigation
- User Story 3, User Navigation
- User Story 4, Merchant Navigation

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

depends on #122

depends on #121

depends on #120

### PR DESCRIPTION
**Pull request by wipegup**
closes #118

### CODE REVIEW

**Review by cnodland — APPROVED** [review]
Looks great, good use of controllers

File: spec/models/order_item_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
     it {should validate_presence_of :item_id}
     it {should validate_presence_of :quantity}
     it {should validate_presence_of :ordered_price}
-    it {should validate_presence_of :fulfilled}
+    it {should validate_inclusion_of(:fulfilled).in_array([true,false])}
```
Comment (Vjp888) [inline spec/models/order_item_spec.rb:9]: Didn't know about this. I like how much more explicit it is on what should be there.

**Review by whois-jon-peterson — APPROVED** [review]
